repose-aggregator/core/valve/pom.xml

```diff
@@ -206,6 +206,12 @@
                                         <exclude>META-INF/*.RSA</exclude>
                                     </excludes>
                                 </filter>
+                                <filter>
```

Comment:
This should have some comments providing context as to why we did this:
https://issues.apache.org/jira/browse/LOG4J2-673 is probably a good place to start

Reasoning why we excluded just the one dependencies plugin.dat is also useful.